chromium_src/chrome/browser/renderer_context_menu/render_view_context_menu.cc

```diff
@@ -320,7 +320,20 @@ void BraveRenderViewContextMenu::InitMenu() {
   // Only show the translate item when go-translate is enabled.
 #if !BUILDFLAG(ENABLE_BRAVE_TRANSLATE_GO)
   index = menu_model_.GetIndexOfCommandId(IDC_CONTENT_CONTEXT_TRANSLATE);
-  if (index != -1)
+  if (index != -1) {
     menu_model_.RemoveItemAt(index);
+
+    // Separator always precedes translate item,
```

Comment:
is this helpful here? It was recently added https://chromium.googlesource.com/chromium/src.git/+/3ac3f99cb82b34690d800a4dfdbefe2630060e39%5E%21/

Reply:
@bridiver , 
https://github.com/brave/brave-core/pull/8914/commits/f507f7b957d590c1a45363baa0204eca425f7040
yes, there are two useful methods: `RenderViewContextMenuBase::RemoveSeparatorBeforeMenuItem` and `RenderViewContextMenuBase::RemoveAdjacentSeparators`.

`RemoveAdjacentSeparators` may be used in the end of `BraveRenderViewContextMenu::InitMenu()` to exclude any possibility of duplicating the separators.

Both `RemoveSeparatorBeforeMenuItem` and `RemoveAdjacentSeparators` have the code in the end
```
  if (toolkit_delegate_)
    toolkit_delegate_->RebuildMenu();
```
But `toolkit_delegate_` is initialized after `BraveRenderViewContextMenu::InitMenu()`, so I had to clear and restore `toolkit_delegate_` to avoid the crash.